libioc/ResourceUpdater.py

```diff
@@ -315,6 +316,13 @@ def fetch(
         self.logger.verbose(
             f"Fetching updates for release '{self.release.name}'"
         )
+
+        symlink_src = self.release.root_path
+        if "p0" in [x.snapshot_name for x in self.release.version_snapshots]:
+            # use p0 snapshot if available
+            symlink_src += "/.zfs/snapshot/p0"
```

Comment:
what else do we use?

Reply:
If there has not been an update yet, no `p0` snapshot exists yet. On a pristine release, we mount the Releases root directory.